elf2flt.c

```diff
@@ -66,6 +66,9 @@ const char *elf2flt_progname;
 #include <elf/m68k.h>
 #elif defined(TARGET_microblaze)
 #include <elf/microblaze.h>
+#if R_MICROBLAZE_max < 33
```

Comment:
is this a define ?  isn't this just an enum ?  and thus this check would always be true (since undefined symbol is 0 which is less than 33) ?

Reply:
Yes, you are right! The good news is that that the #define will work even if it already exists as an enum, but the #if does nothing useful.

Reply:
i'm sure it compiles OK, but it's still confusing/misleading code that i don't think we should merge.

also, should we be supporting non-standard relocs ?  if it's not merged into binutils itself, then there's no guarantee that reloc 33 will be R_MICROBLAZE_32_NONE.

Reply:
It's fine not to merge this. Maybe one day when it's merged into binutil mainline

Reply:
you could test for the define via a configure test, then we wouldn't hardcode the reloc number.  still not great as the reloc hasn't been upstreamed, but it's a bit more palatable, and we can add a comment explaining the origin.

we're doing that for new xtensa relocs in #18 if you want to see one way of doing it.